web/accessibility.md

```diff
@@ -107,11 +126,38 @@ Fact: Most accessibility features are not visible. Many websites and apps are st
 - [Readable fonts](https://thrive.design/best-fonts-for-reading-easiest-to-read-online-design-fonts/)
 - [Tab indexing Mozilla dev docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)
 
+#### Developer/QA Resources
+- [Intro to ARIA](https://www.w3.org/WAI/standards-guidelines/aria/)
+- [WAI-ARIA Authoring Practice 1.1](https://www.w3.org/TR/wai-aria-practices-1.1/)
+- [React Accessibility](https://reactjs.org/docs/accessibility.html)
+- [Automated Accessibility Testing with Axe](https://github.com/dequelabs/axe-core)
+- [Inclusive Components](https://inclusive-components.design/)
+- [Check if your code works with assistive technologies](https://a11ysupport.io/)
+- [Color Contrast Analyzer](https://www.tpgi.com/color-contrast-checker/)
```

Comment:
I said quick reference in the issue, but this is a little too quick.  Could you please also create some more sections for these topics  in "Lesson" of "Independent Practice"?

- A short section answering "What is aria", "why is it important", and that semantics are usually the real answer.
- What is WCAG and its rating system. Lead them in using the Wave browser extension site for 5 minutes - in a on a more accessible site, and a less accessible site.
- A few sentences about zoom (the screen magnification, not the video chat) tech and what websites do to make it work or not work.
- A few sentences about contrast, and lead them in checking an accessible example and an inaccessible example with the contrast checker.


You don't even have to necessarily 'write' these - you can copy-paste from a good resource as long as you cite the article and author + a link, or you can send them to a link, but please tell them exactly which parts to read, and how long it will take them. "Just a link" is far too open ended - learners don't know what the intention is.  They will either never look at it, or get sucked into an internet hole, or anywhere in between.